docs/root/configuration/network_filters/ext_authz_filter.rst

```diff
@@ -0,0 +1,57 @@
+.. _config_network_filters_ext_authz:
+
+External Authorization
+======================
+
+* External authorization service overview :ref:`architecture overview <arch_overview_ext_authz>`
+* :ref:`Network filter v2 API reference <envoy_api_msg_config.filter.network.ext_authz.v2.ExtAuthz>`
+
+The external authorization network filter calls an external authorization service to check if the incoming request is authorized or not.
```

Comment:
nit: please reflow all of this so it's at 100 cols through the paragraph and flows cleanly. Same elsewhere.